page.php

```diff
@@ -0,0 +1,63 @@
+<?php
+get_header();
+?>
+
+<main id="main-content">
+  <section id="posts" class="container">
+
+<?php
+if( have_posts() ) {
+  while( have_posts() ) {
+    the_post();
+
+    $content_en = get_post_meta($post->ID, '_igv_content_en', true);
+    $contact_info = get_post_meta($post->ID, '_igv_contact_info', true);
+?>
+      <article <?php post_class('grid-row margin-top-small margin-bottom-basic'); ?> id="post-<?php the_ID(); ?>">
+        <div class="page-content grid-item item-s-12 item-m-8 active font-size-large margin-bottom-small">
```

Comment:
this content has a 1 column gap between it and the address in the design, but probably best to restrict the `<p>` width with css as doing it with the grid looks complex for midsize screens